cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -193,6 +193,9 @@
     mount options=(rw rbind) {,/usr}/lib{,32,64,x32}/modules/ -> /tmp/snap.rootfs_*{,/usr}/lib/modules/,
     mount options=(rw rslave) -> /tmp/snap.rootfs_*{,/usr}/lib/modules/,
 
+    mount options=(rw rbind) {,/usr}/lib{,32,64,x32}/firmware/ -> /tmp/snap.rootfs_*{,/usr}/lib/firmware/,
+    mount options=(rw rslave) -> /tmp/snap.rootfs_*{,/usr}/lib/firmware/,
```

Comment:
Why are we allowing these to be mounted as read-write? These should be only read-only mounted.

Reply:
Good catch! I think we need to talk to @zyga because e.g. the modules are probably also fine to be mounted RO.

Reply:
I think the mount options are not _super_ accurate as in, they are not really respected the way they should be. I will look at a pass that changes most of them to read only.